ForceBlur/Vendor/UIImageEffects.swift

```diff
@@ -105,8 +105,8 @@ import UIKit
 import Accelerate
 
 public extension UIImage {
-    public func applyLightEffect() -> UIImage? {
-        return applyBlurWithRadius(30, tintColor: UIColor(white: 1.0, alpha: 0.3), saturationDeltaFactor: 1.8)
+    public func applyLightEffect(radius: CGFloat) -> UIImage? {
```

Comment:
In my project I don't want to use default UIBlurEffectStyleLight, so why not another value?